Kuroba/app/src/main/java/com/github/k1rakishou/chan/core/usecase/KurobaSettingsImportUseCase.kt

```diff
@@ -705,6 +706,7 @@ class KurobaSettingsImportUseCase(
       put(6, Wired7.SITE_NAME)
       put(8, Kun8.SITE_NAME)
       put(9, Chan420.SITE_NAME)
+      put(10, Chan370.SITE_NAME)
```

Comment:
You don't need to add the site here because this class is used to import settings from another application (the original fork) and it doesn't support this site so there is simply no point.